src/Filament/Resources/Admin/ForumResource.php

```diff
@@ -16,31 +16,53 @@
 use Filament\Tables\Columns\TextColumn;
 use Filament\Tables\Filters\SelectFilter;
 use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Model;
 use Tapp\FilamentForum\Filament\Resources\Admin\ForumResource\Pages\CreateForum;
 use Tapp\FilamentForum\Filament\Resources\Admin\ForumResource\Pages\EditForum;
 use Tapp\FilamentForum\Filament\Resources\Admin\ForumResource\Pages\ListForums;
 use Tapp\FilamentForum\Models\Forum;
-use UnitEnum;
 
 class ForumResource extends Resource
 {
     protected static ?string $model = Forum::class;
 
-    protected static string|UnitEnum|null $navigationGroup = 'Forums';
-
     protected static string|BackedEnum|null $navigationIcon = 'heroicon-o-rectangle-stack';
 
     protected static ?string $slug = 'forums';
 
+    public static function getNavigationGroup(): ?string
+    {
+        return __('filament-forum::filament-forum.admin.navigation-group');
```

Comment:
```suggestion
        return __('filament-forum::admin.navigation-group');
```

I don't think we need both `filament-forum` here, or any of these references.